### PR TITLE
Added more Info's for contributors

### PR DESCRIPTION
**Pull request by i-walker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,36 @@ If it is inside an interface or class, use the method, i.e.
...
+```markdown
+```kotlin:ank:playground
+import arrow.data.ForListK
+import arrow.data.ListK
+import arrow.Kind
+import arrow.data.fix
+import arrow.typeclasses.Functor
+import arrow.extension
+
+@extension
```
Comment (raulraja) [inline README.md]: Ank can't expand annotations yet.

File: README.md [hunk 2]
```diff
@@ -38,6 +38,36 @@ If it is inside an interface or class, use the method, i.e.
...
+interface ListKFunctor : Functor<ForListK> {
+    override fun <A, B> Kind<ForListK, A>.map(f: (A) -> B): ListK<B> =
+        fix().map(f)
+}
+```... 
+```
+[Playground Docs](https://github.com/arrow-kt/arrow-playground).
+But for now a basic set-up as the snippet above is sufficient. Remember that ank:playground wont make the code snippet global for other snippets.
+
+- Sometimes Ank won't process very long (over ~160 lines) md files. Split them up into < chapter >-Part1.md , < chapter >-Part2.md
```
Comment (raulraja) [inline README.md]: When we've seen this in the past it's usually related to the use of coroutines in the examples. So etching the script engine chokes with resetting internally it's stack variables

**Review by raulraja — APPROVED** [review]
Let's remove the part about @extensio  with ank or clarify it and this is good to go.

File: README.md (hunk 2 above)
Comment (i-walker) [reply README.md]: So should I just add your Comment?

**Pull request merged**